### PR TITLE
PD-6962 Added health endpoint

### PR DESCRIPTION
**Pull request by rvanburen-r7**
health HTTP API endpoint

### CODE REVIEW

File: lib/control/v1/health.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    res.set('Allow', allowedMethods);
+    res.status(HTTP_METHOD_NOT_ALLOWED);
+    res.end();
+  };
+
+  routes.health.get((req, res) => {
+
+    res.status(HTTP_OK).json({
+
+      status: 'okay',
```
Comment (dgreene-r7) [inline lib/control/v1/health.js]: Is this always going to return `okay` and `200`? Is there a situation where warden might not be ok?

Comment (rvanburen-r7) [inline lib/control/v1/health.js]: The only thing I can think of would be if Warden couldn't contact the vault cluster. but seeing as it is connecting to local host, do we need another check for that?

Comment (dgreene-r7) [inline lib/control/v1/health.js]: I think it brings up a greater question as to how you test the health of an effectively stateless service.

To my mind it's a good endpoint to hit every _n_ seconds to make sure the service is running.

Comment (rvanburen-r7) [inline lib/control/v1/health.js]: This test would be done on a different service though, right? Warden shouldn't be checking it's own health to make sure it's healthy.

**Comment by dgreene-r7** [comment]
👍

**Pull request merged**